egs/iam/s5/local/unk_arc_post_to_transcription.py

```diff
@@ -51,33 +76,34 @@
   word = line_vect[4]
   phones = line_vect[5]
```

Comment:
In order to make sense of this part you have to explain the format of the 1best-arc-post file at the beginning.